randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3012,8 +3031,6 @@ Extra - asset_id: collision_camera_036
       Any of the following:
           Morph Ball
           After Cataris - breakablemag008 and Can Slide
-  > Event - Top Blob (breakablemag007)
-      Shoot Beam
```

Comment:
why remove this? you set it as diffusion or wave before, which is fine.

Reply:
Was a misunderstanding on my part, it has been re-added, my understanding of events is still coming along